package foo

func foo() (err error) {
	return
}
